yt_dlp/extractor/ard.py

```diff
@@ -360,11 +361,29 @@ def _extract_episode_info(self, title):
     def _real_extract(self, url):
         display_id = self._match_id(url)
 
-        page_data = self._download_json(
-            f'https://api.ardmediathek.de/page-gateway/pages/ard/item/{display_id}', display_id, query={
-                'embedded': 'false',
-                'mcV6': 'true',
-            })
+        token = self._download_json('https://sso.ardmediathek.de/sso/token', display_id)
```

Comment:
when the cookie is not set, this fails, so I need a parameter here to ignore failures

ERROR: [ARDMediathek] Y3JpZDovL2Rhc2Vyc3RlLmRlL29kZXJicnVjaC8yMDI0LTAxLTIwXzAwLTAwLU1FWg: Unable to download JSON metadata: HTTP Error 400: Bad Request (caused by <HTTPError 400: Bad Request>)